scripts/process/process.storm_track.R

```diff
@@ -0,0 +1,16 @@
+
+process.storm_track <- function(viz = as.viz('storm-track')){
+  library(rgeos)
+  library(sp)
+  depends <- readDepends(viz)
+  view.lims <- depends[['view-limits']]
+  track <- depends[[2]]
```

Comment:
I don't know depends well, but is it safe to use numerical indexing?

Reply:
I will change that